### PR TITLE
Add rules for Wikiscan.org

### PR DESCRIPTION
**Pull request by Nintendofan885**
Add rules to redirect http://(&#42;).wikiscan.org -> https://(&#42;).wikiscan.org

### CODE REVIEW

File: src/chrome/content/rules/Wikiscan.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<ruleset name="Wikiscan">
+	<target host="*.wikiscan.org" />
+        <target host="wikiscan.org" />
+  
+    <rule from="^http:" to="https:" />
+<test url="http://en.wikiscan.org/" />
+<test url="http://frwiktionary.wikiscan.org/" />
+<test url="http://eswikisource.wikiscan.org/" />
+</ruleset>
```
Comment (pipboy96) [inline src/chrome/content/rules/Wikiscan.xml:9]:
```suggestion
<ruleset name="Wikiscan">
	<target host="wikiscan.org" />
	<target host="*.wikiscan.org" />

	<rule from="^http:" to="https:" />

	<test url="http://en.wikiscan.org/" />
	<test url="http://frwiktionary.wikiscan.org/" />
	<test url="http://eswikisource.wikiscan.org/" />
</ruleset>
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]

**Comment by Nintendofan885** [comment]
You don't want to know how angry I am that that was the reason why chrome was failing the check

**Comment by pipboy96** [comment]
@Nintendofan885 Actually, it wasn't. This check often fails without any detectable reason. In this case, close and re-open the PR to restart the check.

**Review by pipboy96 — APPROVED** [review]

**Comment by Nintendofan885** [comment]
> @Nintendofan885 Actually, it wasn't. This check often fails without any detectable reason. In this case, close and re-open the PR to restart the check.

Oh ok. It just confused me as it suddenly passed the check after I accepted your changes

**Review by cschanaj — APPROVED** [review]

**Comment by cschanaj** [comment]
thanks!

**Comment by Nintendofan885** [comment]
No problem

**Pull request merged**